test/uk/gov/hmrc/tai/viewModels/TaxCodeViewModelSpec.scala

```diff
@@ -48,6 +59,13 @@ class TaxCodeViewModelSpec extends PlaySpec with FakeTaiPlayApplication with I18
         result.ledeMessage mustBe Messages("tai.taxCode.single.info")
         result.title mustBe s"${Messages("tai.taxCode.single.code.title.pt1")} $taxYearSuffix"
       }
+
+      "provided with valid input for a previous tax year" in {
+        val result = previusYearSUT(Seq(taxCodeIncomes1))
```

Comment:
Can this be called something more meaningful than SUT?

Reply:
Agreed, it's done now